---
output: html_document
editor_options: 
  chunk_output_type: console
---
# Preface {.unnumbered}

> The cover picture of this book was generated by Gemini, Large language model. Google 2024.

## WP4 of INIKA Project

Work package leader for WP4: 
- Dr. Eve Fiskebeck (NVI) 
- Dr. Elpidius Rukambile (TVLA)

Project leader: 
- Prof Madelaine Norström

In the present document (index/preface), are the plans for WP4. 
The other parts of the book, will cover data analysis code, 
necessary information on methods and analysis to prepare puclications, 
and results.

## Objectives (Pasted from the Project description)

- T4.1. Merge and assess the quality of data collected through WP1 to WP3. D.4.1 A clean data “set” for further analyses in T.4.2-T.4.4.

- T4.2.Determine possible associations between drivers such as AMU/attitudes and the phenotypic resistance of bacteria within and between sectors using logistic regression models. Possible differences in AMR and AMU according to geographical area and season will also be assessed. D.4.2 Peer review scientific publications and/or proceedings.

- T4.3 Assess the patterns of AMR determinants (i.e. genes, transferable plasmids) in the three sectors using statistical modelling. The number of isolates obtained from each source (human, animal and environment) and the number of isolates where the WGS analyses have been performed determines what will be included in the statistical analyses. D.4.3 Peer review scientific publications and/or proceedings.

- T4.4 Comparative genomics of bacterial isolates from different sources and sectors, and use a “OH” approach to elucidate associations between reservoirs. D.4.2 Per review scientific publications and/or proceedings.

## Tasks - Working plan (& Status)

### T4.0 Genera functionning of WP4
- [ ] decide on when to have first meeting with Dr Elpidius Rukambile. First meeting 
with Madelaine for me will be 17/03 for both start WP4 and Students. Should we do
that short after ? 
- [ ] need to implement regular meetings for progress

### T4.1 
- [ ] File: `data_description/data_information.qmd`. Short presentation of the datasets sources. Including notes of collections that are important to consider for analyses


- [ ] File: `data_description/data_wrangling.qmd`. 
  - [ ] Assessing data quality (individually for each datgaset) (might want to consider splitting in subparts one per dataset) 
  - [ ] Eventual feedback on error detected and corrections if necessary.
  - [ ] Dataset merging & quality check (globally)
  

- [ ] File: `data_description/data_description.qmd`.
  - [ ] Basic statistics merged dataset (getting an overview of what we have currently
  to be able to discuss)
  
- [ ] Discussion together : 
    - further check? 
    - correction of datasets ? 
    - do we know what we can obtain from the dataset ? 

### T4.2 

- [ ] Eve Refreshing stats : Logistic regression (in parallel maybe course material)

Determine possible associations between drivers such as AMU/attitudes and the phenotypic resistance of bacteria within and between sectors using logistic regression models. Possible differences in AMR and AMU according to geographical area and season will also be assessed. D.4.2 Peer review scientific publications and/or proceedings.

### T4.3
### T4.4


# 2025-03-17 - meeting with madelaine (notes - to reorganize)

Things are in K ...

\\vetinst.no\dfs-felles\Felles02\FAG\Tverrfaglig\AMR\FoU-aktiviteter & prosjekter\31218_INIKA_OH_TZ\Working plans



- occurence and distribution of genes ...
- [ ] ... genes vs gene mutations ? 

- When is the final data coming ?  in september
- when is the dry season data with the bacteria - it should be comming soon - latest in 
july ... for sequencing
--- elpidius need to have a plan


Will decide what will be sequenced based on AMR profiles ... 
Genetic analyiss
1. confirm species identification
2. determine resistance genes


Data sets 
- 1. KAP study - WP4 - Kobo Toolbox 
-- 253 results - Insitutional capacity ... -> we will use paractice how often osv ...
K/ INIKA / PhD students / meanuel ... latest 
> list of substances for the dataset - was not cleaned ... Emanuel has to clean ... 

- [ ] develop the code for cleaning the data and checking the data 

